README.md

```diff
@@ -20,12 +20,12 @@ Note that Asymptote compilation is done remotely, so Asymptote does not need to
 
 Configuration:
 
-- In the `project.ptx` file, change the URL for the WeBWorK server to your local server
-- In `publication/publication.ptx`, change the `baseurl` to the URL where your copy will be hosted
+- In `publication/publication.ptx`, change the `baseurl` to the URL where your copy will be hosted, and set the `webwork` server to your local server.
+- If you need a newer version of Pretext than included with PreTeXt CLI (or wish to set a custom stylesheet), add `<xsl>../pretext/xsl/pretext-html.xsl</xsl>` tags to `project.ptx` targets.
```

Comment:
replace "Pretext" with `<pretext/>`

Reply:
This is markdown, so I think you mean just do the capitalization thing on L24? "PreTeXt" ?

Reply:
Oh yes, of course. Yes, ideally the correct capitalisation should be used.